group/init/README.md

```diff
@@ -27,7 +27,7 @@ because the definition of `group_id` depends on the key of this message, which w
 be known until this is published.
 
 This means this initial message and it's content will need to be **manually boxed**,
-with the only `recipient_key` being the symmetric `group_key` for this new group.
+with the `recipient_key`s being the symmetric `group_key` for this new group as well as to your `own_key`, for recovery purposes.
```

Comment:
Confusing that we'd encrypt with the own_key for recovery. It does help us if the only thing we'd lost is the group_key, but what if we lose the own_key? Since of what I understand the own_key isn't directly or indirectly backed up by e.g. your recovery phrase in manyverse.

Maybe a discussion for a different time though.

Reply:
> Since of what I understand the own_key isn't directly or indirectly backed up by e.g. your recovery phrase in manyverse.

No, it would be backed up by the 24 words. I made some changes in ssb-keyring specifically so that all the keys there would be recoverable from 24 words.